SAMPLES.md

```diff
@@ -26,32 +26,31 @@ The minimum requirements for UIKit for React are:
 
 - React 16.8.0+
 - React DOM 16.8.0+
-- Sendbird Chat SDK for JavaScript 3.0.115+
+- @sendbird/chat v4
 - css-vars-ponyfill 2.3.2
 - date-fns 2.16.1
 
 ### Implement Chat with App component
 
-The `App` component is a collection of all UIKit components you need to implement chat. This is included in all core component samples, so be sure to set your own APP_ID, USER_ID, and NICKNAME in  `const.js` in each to customize your sample. On the [CodeSandbox](https://codesandbox.io/s/1-1-using-sendbird-app-9xum5) link, you will see that the props of the `App` component refer to use the values of the correspondings of `const.js` for initialization.
+The `App` component is a collection of all UIKit components you need to implement chat. This is included in all core component samples, so be sure to set your own APP_ID, USER_ID, and NICKNAME in  `const.js` in each to customize your sample. On the [CodeSandbox](https://codesandbox.io/s/1-1-using-sendbird-app-9l4d4b?file=/src/App.js) link, you will see that the props of the `App` component refer to use the values of the correspondings of `const.js` for initialization.
```

Comment:
I think it's better to use `https://codesandbox.io/s/1-1-using-sendbird-app-9l4d4b`
because file name or location can be changed by humans hand